primary/src/block_synchronizer/handler.rs

```diff
@@ -151,6 +167,40 @@ impl BlockSynchronizerHandler {
 
 #[async_trait]
 impl Handler for BlockSynchronizerHandler {
+    /// This method kicks start a process to fetch digests of missing certificates after the latest local round.
+    /// Then certificates are fetched and synchronized for these digests, from the lowest to the highest round.
+    /// The goal is to allow fast catch up of a validator lagging behind, by reducing the need to use the slow
+    /// recursive causal completion code path to synchronize certificates.
+    #[instrument(level = "trace", skip_all)]
+    async fn synchronize_range(&self) -> Result<(), Error> {
+        self.metrics
+            .range_sync_started
+            .with_label_values(&[&self.committee.epoch.to_string()])
+            .inc();
+
+        let (tx_certificate_ids, mut rx_certificate_ids) = channel(1);
+        self.tx_block_synchronizer
+            .send(Command::SynchronizeRange {
+                respond_to: tx_certificate_ids,
+            })
+            .await
+            .expect("Couldn't send SynchronizeRange message to block synchronizer");
+        let certificate_ids = rx_certificate_ids.recv().await.unwrap_or_default();
```

Comment:
What is the default case here? I am referring to `unwrap_or_default()`. Simply an empty list right?